src/corehost/packaging/Microsoft.NETCore.App/project.json

```diff
@@ -0,0 +1,46 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160321-04",
+    "Microsoft.CodeAnalysis.VisualBasic": "1.3.0-beta1-20160321-04",
+    "Microsoft.CSharp": "4.0.1-rc2-23931",
+    "Microsoft.NETCore.DotNetHostPolicy": "1.0.1-beta-002200",
```

Comment:
As we rev the version of this package from the CLI, how can we update the dependency of the metapackage as well?